### ISSUE TITLE
rename HexagonOffsetGrid to HexagonGridView?

### ISSUE THREAD

**Issue by slovnicki** [rSquared-software/flutter-hexagon#3]
What do you think of that, to be more in line with Flutter naming for [GridView](https://api.flutter.dev/flutter/widgets/GridView-class.html).

Maybe some efforts can also be put into making the interface and API very similar to `GridView`. Just some thoughts...

**Comment by orlikraf** [comment]
I don't think it is necessary to add 'View' onto already long widget name. I'm developing another hexagon grid which will be more hexagonal in shape so `HexagonGrid` is already reserved for it. 

About improving API what elements of `GridView` do you think are needed?

**Comment by slovnicki** [comment]
@orlikraf I don't think `HexagonGridView` is too long. It even has fewer letters than `HexagonOffsetGrid` :)

> About improving API what elements from GridView do you think are needed?

I would have to spend more time exploring your package to answer this, but I will definitely do that in the upcoming days as I have quite a use for this package and want to understand it better.

On a side note: is it possible to add border to `HexagonWidget`? (separate issue can be created for this if it's not implemented yet)

**Comment by orlikraf** [comment]
> is it possible to add border to HexagonWidget?
It isn't right now. I have borders on my mind. For the time being you have to prepare a background with border or nest hexagon with content in a bigger one that will be acting as a border. 
Perfect solution would allow to set border sides separately.
